frontend/src/components/LocalhostScanForm/LocalhostScanForm.tsx

```diff
@@ -28,42 +29,53 @@ export const LocalhostScanForm: React.FC<Props> = ({ setSelected }) => {
     emit('scan', 'localhost')
   }, [])
 
-  const updateTargets = useCallback(() => {
-    const selected = scanData.filter((_, key) => state[key])
-    setSelected(selected)
-  }, [scanData, setSelected, state])
+  const updateTargets = useCallback(
+    checked => {
+      const selected = scanData.filter((_, key) => checked[key])
+      console.log('selected', selected)
```

Comment:
Do you mean to keep this console.log here?

Reply:
Nope good catch.